src/ert/run_models/single_test_run.py

```diff
@@ -16,9 +16,9 @@
 class SingleTestRun(EnsembleExperiment):
     """
     Single test is equivalent to EnsembleExperiment, in that it
-    samples the prior and evaluates it. There are two key differences:
-    1) Single test run always runs locally using the local queue
-    2) Only a single realization is run
+    samples the prior and evaluates it.<br>There are two key differences:<br>
+    1) Single test run always runs locally using the <b>local queue</b><br>
+    2) Only a <b>single realization</b> is run<br>
```

Comment:
Could we also indicate that it is always realization 0 that is being run? (it matters for those running design matrices)

Reply:
Sure thing!

Reply:
<img width="587" alt="Screenshot 2024-09-27 at 15 38 53" src="https://github.com/user-attachments/assets/9caf058b-8207-4458-8ace-bfc2b5c5e26f">

If you have a proposal, I'm all ears. 👂